README.md

```diff
@@ -10,26 +10,28 @@ See: https://raw.github.com/RIPE-NCC/whois/master/LICENSE.md
 Running Integration Tests for Development
 -----------------------------------------
 
-It is preferable to write integration tests during development, and commit them along with any code changes. All dependencies apart from the database are mocked. 
+It is preferable to write integration tests during development, and commit them along with any code changes. All dependencies apart from the database are mocked.
 
 Running Whois Locally For Development
 -------------------------------------
 
 You can run a standalone Whois server locally. Dependencies must be configured in a local properties file.
```

Comment:
i was told not to do this because there's quite a lot to change to make sure it doesn't interfere with other environments. in fact, before i was told *not* to do this i followed the README and ended up sending weird stuff to Sentry which confused Ed and Mahesh.

Reply:
oh...ok, Ed told me to do this, it is in the task :S 
I will talk tomorrow with Mahesh then to figure out how to do